tests/webserver.py

```diff
@@ -32,7 +33,7 @@ def __init__(self, host="", port=8000):
         self.thread = threading.Thread(target=self.server.serve_forever, daemon=True)
 
     @property
-    def host(self):
+    def host(self) -> Any | str:
```

Comment:
Is this really Any or string? Why not just a string or int?

Reply:
it has the types "str | bytes | bytearray" leaving it at "str" or "str | int" would show an error. But on tracing the variables and method calls it shows type assigned as "_RetAddress" which was used as a type alias for "Any".

Reply:
Ah okay gotcha, thanks for that explanation.